src/Test/test_solve.jl

```diff
@@ -1423,7 +1423,7 @@ function test_solve_conflict_zeroone_ii(
     c2 = MOI.add_constraint(
         model,
         MOI.ScalarAffineFunction(MOI.ScalarAffineTerm.(one(T), [x]), zero(T)),
-        MOI.EqualTo(div(one(T), T(2))),
+        MOI.EqualTo(one(T) / T(2)),
```

Comment:
This test only makes sense if `(one(T) / T(2)) isa T` right?

Reply:
correct!